cmd/aida-rpc/run_rpc.go

```diff
@@ -32,6 +32,10 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	RpcDefaultProgressReportFrequency = 100_000
```

Comment:
Since this const is used only in this file, it doesn't need to be a public variable.
```suggestion
	rpcDefaultProgressReportFrequency = 100_000
```